libs/arcade-serve/arcade_serve/fastapi/telemetry.py

```diff
@@ -42,7 +44,10 @@ def instrument_app(self, app: FastAPI) -> None:
                 "🔎 Initializing OpenTelemetry. Use environment variables to configure the connection"
             )
             self.resource = Resource(
-                attributes={SERVICE_NAME: "arcade-worker", "environment": self.environment}
+                attributes={
+                    service_attributes.SERVICE_NAME: "worker",
+                    deployment_attributes.DEPLOYMENT_ENVIRONMENT: self.environment,
```

Comment:
This is the new, non-deprecated version that datadog supports: 

```suggestion
                    deployment_attributes.DEPLOYMENT_ENVIRONMENT_NAME: self.environment,
```